### PR TITLE
#38 - composer binary improvements

### LINKED ISSUE #38: Change Composer to binary
I would like to switch current Composer installation to the one described here: https://github.com/composer/docker/pull/250

### PR DESCRIPTION
**Pull request by mtracz**
Use composer binary from docker image instead of curl installation.

Should close #38

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,18 @@
 ARG PHP_VERSION=8.1.12-fpm-alpine
+ARG COMPOSER_TAG_VERSION=2-bin
```
Comment (krzysztofrewak) [inline Dockerfile:2]:
```suggestion
ARG COMPOSER_TAG_VERSION=2.4.4-bin
```
Why not exact version?
ref: https://hub.docker.com/layers/composer/composer/2.4.4-bin/images/sha256-1fcc14e3597f38f171393b91f0a16e55aac38d1479e44368669da93c4d0ddc13?context=explore

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
Have you checked how smaller is this new container sizewise?

File: Dockerfile (hunk 1 above)
Comment (mtracz) [reply Dockerfile:2]: Didn't notice that this tag is available.

**Comment by mtracz** [comment]
Final image size is the same (99,7 MB) for both cases. 
Build with buildx, without cache is 2,5 seconds faster with copying composer from the image.
After cache layers, the build is the same fast.

To sum up, there is no big difference, but Dockerfile is more clear when composer bin is copying from the image.

**Review by krzysztofrewak — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**